### DISCUSSION TITLE
Works until field for working contracts in case employee leaves the company

### DISCUSSION THREAD

**Discussion by habibhaidari1** [kimai/kimai#4418] [Plugins]
Hello, 
I really like the new controlling plugin, but it needs an additional date field ("works until") for off-boarding employees. This field should be used when the employee leaves the company. As a result, the controlling plugin should stop counting the working hours from this date. 

Is there any intention to get this functionality or is there a clever way to stop the counter

Kindly regards

**Comment by kevinpapst** [comment]
Thanks for the idea, never thought about it, but makes sense. I added it to the list of ideas.
